utilities/validations.py

```diff
@@ -24,6 +24,8 @@ def validate_empty_fields(**kwargs):
         value = kwargs.get(field)
         if not type(value) is bool and not value:
             raise AttributeError(field + " is required field")
+        if isinstance(value, str) and not value.strip():  # remove white space
+            raise AttributeError(field + " cannot be empty")
```

Comment:
The message `can not be empty` and `required field`  Kind of look similar to me. We could use a better message here or use the same message for both by refactoring this code like this:
```
for field in kwargs:
        value = kwargs.get(field)
        if isinstance(value, str):
            value = value.strip()
        if not type(value) is bool and not value:
            raise AttributeError(field + " is required field")
```

Reply:
Kindly fix the code climate issue too

Reply:
@nakatuddesuzan`required field` validates the existence of a field. On the other hand, empty validates if it has a value i.e it doesn't contain white spaces only. Note: a field can exist and be empty  but cannot be non-existent and be empty.

Reply:
Hello @verenceLola ,  I get the point.  how about we use `Spaces are not allowed!` for the `RegularExpressionValidator` and `field cannot be empty` or `field is required` for `RequiredFieldValidator`. The latter 2 messages mean the same.

Reply:
Hey @verenceLola, The message `cannot be empty` though I still feel like those two instances should display the same message since the validation is similar. Empty space means there is no string in the field hence that field is required.

Reply:
I think @nakatuddesuzan suggestion is better, `""` and `"  "` should get the same message.

Reply:
good work @verenceLola though I request that we have the same message as mentioned above.